src/dbos-runtime/config.ts

```diff
@@ -69,6 +74,14 @@ export function loadConfigFile(configFilePath: string): ConfigFile {
     const configFileContent = readFileSync(configFilePath);
     const interpolatedConfig = substituteEnvVars(configFileContent);
     const configFile = YAML.parse(interpolatedConfig) as ConfigFile;
+    if (!configFile) {
+      throw new Error(`${configFilePath} is empty `);
+    }
+    const schemaValidator = ajv.compile(dbosConfigSchema);
+    if (!schemaValidator(configFile)) {
+      const errorMessages = prettyPrintAjvErrors(schemaValidator);
+      throw new Error(`${configFilePath} failed schema validation. ${errorMessages}`);
+    }
```

Comment:
I trust this makes for a much clearer experience: if you have an invalid config file, throw early with an explicit message.